ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -34,6 +34,42 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
         )
   }
 
+  object CommandErrors extends ErrorGroup() {
+
+    @Explanation("This rejection is given when a command submission could not be processed.")
+    @Resolution("Refer to the error details and/or retry.")
+    object CommandSubmissionFailure
+        extends ErrorCode(
+          id = "COMMAND_SUBMISSION_FAILURE",
+          ErrorCategory.ContentionOnSharedResources,
+        ) {
+      case class Reject(messagePrefix: String, override val throwableO: Option[Throwable])(implicit
```

Comment:
As I see, this rejection appears in two situations:
* On unhandled/unexpected exception when offering to the queue in the `QueueBackedTracker`. In that situation, the error is serious and should be `INTERNAL` (SystemInternalAssumptionViolated).
* On queue closed. We don't have a perfect category for it, but based on the Grpc status codes, I would use `TransientServerFailure` (`UNAVAILABLE`). I think you can re-use the `ServiceNotFound` error for this one.

Reply:
1) Ok
2) I can't seem to find `ServiceNotFound` among error codes. Where is it?

Reply:
Sorry, it was late when I did this review. I meant `ServiceNotRunning`